src/app/services/api.service.ts

```diff
@@ -1,9 +1,48 @@
 import { Injectable } from '@angular/core';
+import { CardComponent} from "../card/card.component";
+
 
 @Injectable({
   providedIn: 'root'
 })
 export class ApiService {
 
-  constructor() { }
+  url = "http://localhost:3000/tasks";
+  catTipsUrl = "https://catfact.ninja/fact";
+
+  async getAllTasks(): Promise<CardComponent[]> {
```

Comment:
The Promise type should not be the `CardComponent` type, it should be the interface `Task[]`, that interface should be already in main, so make sure to rebase your branch with main

Reply:
Same for the rest of methods that use `CardComponent` as type